src/ape/_cli.py

```diff
@@ -28,13 +29,19 @@ def display_config(ctx, param, value):
 class ApeCLI(click.MultiCommand):
     _commands = None
 
+    def invoke(self, ctx: click.Context) -> Any:
+        try:
+            super().invoke(ctx)
```

Comment:
because of the return `Any`, should this statement be:
```suggestion
            return super().invoke(ctx)
```